docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
       - 8080:8080
       - 5001:5001
   vitalam-node:
-    image: ghcr.io/digicatapult/vitalam-node:latest-dev
+    image: ghcr.io/digicatapult/vitalam-node:v2.4.0
```

Comment:
For now this is fine. Can you make a ticket to cover fixing this properly if you haven't already?

Reply:
https://digicatapult.atlassian.net/browse/IN-65